apps/frontend/src/components/global/upload_tabs/LoadFileList.vue

```diff
@@ -152,6 +163,33 @@ export default class LoadFileList extends Vue {
     this.deleteItemDialog = true;
   }
 
+  onCopy() {
+    SnackbarModule.notify('Successfully copied share link');
+  }
+
+  onCopyFailure() {
+    SnackbarModule.failure('Failed to copy to your clipboard');
+  }
+
+  joinEvaluationsByIds(evaluations: IEvaluation[] | Sample[]): string {
+    let stringresult = '';
+    evaluations.forEach((evaluation) => {
+      if(evaluation.hasOwnProperty('id')){
+        stringresult += `${(evaluation as IEvaluation).id},`;
+      }
+    });
+    return stringresult.slice(0, -1);
+  }
+
+  shareItems(evaluations: IEvaluation): string {
+    if (this.selectedFiles.length >= 1){
+      const shareList = this.joinEvaluationsByIds(this.selectedFiles)
+      return `${ServerModule.externalURL || window.location.origin}/share/${shareList}`;
+    } else {
+      return `${ServerModule.externalURL || window.location.origin}/share/${evaluations.id}`;
+    }
```

Comment:
Any chance of being able to leverage https://router.vuejs.org/guide/essentials/passing-props.html#boolean-mode instead and just adding a route such as `/results/1,2,3,4,5` or something? It would alleviate the need for using cookies for this one, and just allow the frontend router to handle it (I assume the frontend router is working in the Docker container).